packages/upload-api/src/types.ts

```diff
@@ -372,37 +378,27 @@ export interface DudewhereBucket {
 }
 
 export interface StoreTable {
-  inspect: (link: UnknownLink) => Promise<StoreGetOk>
+  inspect: (link: UnknownLink) => Promise<StoreInspectOk>
   exists: (space: DID, link: UnknownLink) => Promise<boolean>
-  get: (space: DID, link: UnknownLink) => Promise<StoreAddOutput | undefined>
+  get: (space: DID, link: UnknownLink) => Promise<StoreGetSuccess | undefined>
   insert: (item: StoreAddInput) => Promise<StoreAddOutput>
   remove: (space: DID, link: UnknownLink) => Promise<void>
   list: (
     space: DID,
     options?: ListOptions
-  ) => Promise<ListResponse<StoreListItem>>
-}
-
-export interface TestStoreTable {
-  get(
-    space: DID,
-    link: UnknownLink
-  ): Promise<
-    (StoreAddInput & StoreListItem & { insertedAt: string }) | undefined
-  >
+    ) => Promise<ListResponse<StoreAddInput & StoreListItem>>
 }
 
 export interface UploadTable {
-  inspect: (link: UnknownLink) => Promise<UploadGetOk>
+  inspect: (link: UnknownLink) => Promise<UploadInspectOk>
```

Comment:
Would be nice if these were named consistently like the others...

```suggestion
  inspect: (link: UnknownLink) => Promise<UploadInspectSuccess>
```